package.json

```diff
@@ -270,6 +270,7 @@
     "**/socket.io-parser": "4.0.5",
     "**/ua-parser-js": "0.7.33",
     "browserify-sign": "4.2.2",
+    "devtools-protocol": "0.0.1346313",
```

Comment:
Added this to resolutions so that dependencies that have `devtools-protocol` (specifically `chrome-remote-interface`) will resolve to this version. Without this, there were type conflicts between `devtools-protocol` 0.0.13.. types and `chrome-remote-interface/node_modules/devtools-protocol 0.0.9`.. types.